### PR TITLE
 Tom can now comment on and approve pull requests

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a PR! Maybe @vpodzime can review this?

File: tom.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+
+class GitHub():
+    def __init__(self, token):
+        self.token = token
+        self.headers = {"Authorization": "token {}".format(token), "User-Agent": "cf-bottom"}
+        self.get_cache = {}
+
+    def path(self, path):
+        if path[0] == "/":
```
Comment (vpodzime) [inline tom.py]: ``path.startswith("/")``

File: tom.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+        # print(r.status_code)
+
+        assert r.status_code >= 200 and r.status_code < 300
+        data = r.json()
+        # print(pretty(data))
+        self.get_cache[path] = data
+        return data
+
+    def put(self, path, data):
+        pass
```
Comment (vpodzime) [inline tom.py]: ``raise NotImplemented`` maybe?

File: tom.py [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        self.repos = self.github.get("/user/repos")
+        self.repos += self.github.get("/orgs/cfengine/repos")
+
+        self.pulls = []
+        for repo in self.repos:
+            print("Fetching pull requests for {}".format(repo["full_name"]))
+            pulls = self.github.get(repo["url"] + "/pulls")
+            if pulls:
+                for pull in pulls:
+                    self.pulls.append(pull)
```
Comment (vpodzime) [inline tom.py]: ``self.pulls.extend(pulls)``

**Comment by olehermanse** [comment]
@vpodzime Please review again. Commit history is intact so you can start where you left off.

**Pull request merged**